app/views/external_users/advocates/claims/_basic_and_fixed_fees_form_step.html.haml

```diff
@@ -1,3 +1,7 @@
+-# Is this page still in use?
```

Comment:
Is it?

Reply:
as far as I can tell - I don't think so..